contracts/SimpleSwap.sol

```diff
@@ -32,15 +30,9 @@ contract SimpleSwap {
     uint decreaseTimeout; /* issuer has to wait decreaseTimeout seconds after decrease is requested to decrease hardDeposit */
     uint canBeDecreasedAt; /* point in time after which harddeposit can be decreased*/
   }
-  /* structure to keep track of the latest cheque for one beneficiary */
-  struct ChequeInfo {
-    uint serial; /* serial of the last submitted cheque */
-    uint amount; /* cumulative amount of the last submitted cheque */
-    uint paidOut; /* total amount paid out */
-    uint cashTimeout; /* timeout after which payout can happen */
-  }
-  /* associates every beneficiary with their ChequeInfo */
-  mapping (address => ChequeInfo) public cheques;
+
+  /* associates every beneficiary with their paidOutCheques */
+  mapping (address => uint) public paidOutCheques;
```

Comment:
Should we perhaps rename this to `paidOutAmount` or something?

Reply:
Good idea, what about just `paidOut`?

Reply:
That's fine too